client/src/components/Nav/Nav.js

```diff
@@ -17,10 +17,10 @@ class Nav extends React.Component {
           <div className="links">
           <Link to="/"><b>Home</b></Link>
```

Comment:
Let's remove bold tag in favor of CSS font-weight, more maintainable and less code.